cmd/validator/validator.go

```diff
@@ -105,6 +107,12 @@ func getFlags() (validatorConfig, error) {
 		return validatorConfig{}, errors.New("Wrong parameter value for reporter, groupby is not supported for JUnit reports")
 	}
 
+	if *reportTypePtr == "sarif" && *groupOutputPtr != "" {
```

Comment:
Could you combine this with the Junit check above?

Reply:
done